learning_journal/templates/home.jinja2

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
```

Comment:
See Cris's above comment about lines of code before the "extends" portion. These lines might be why your site doesn't work.